php/system.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+$PI_HOLE_DIR = "/etc/pihole";
+
+$ENABLE_STUBS = true;
+
+if ($ENABLE_STUBS) {
+    $PI_HOLE_DIR = "stubs";
+}
+
+function isUsingStubs() {
+    global $PI_HOLE_DIR;
+    return $PI_HOLE_DIR == "stubs";
+}
+
+function getFileList($type) {
+    global $PI_HOLE_DIR;
+    return file_get_contents($PI_HOLE_DIR . "/" . $type . "list.txt");
```

Comment:
It would be more secure to limit the files it can get to only `whitelist.txt` or `blacklist.txt`, so there's no possibility for exploitation.